app/src/main/java/com/douglasbruce/tasky/MainActivityViewModel.kt

```diff
@@ -40,4 +43,10 @@ class MainActivityViewModel @Inject constructor(
             _isLoading.value = false
         }
     }
+
+    fun logout() {
+        runBlocking {
+            sessionManager.logout()
+        }
+    }
```

Comment:
How comes, you're using `runBlocking` here vs. `viewModelScope`?

Reply:
Not too sure why. Will update to use `viewModelScope`